.github/workflows/nightly-build.yml

```diff
@@ -44,8 +45,8 @@ jobs:
     - name: Build with Maven
       run: mvn -B clean install -U -Pintegration
     - name: Build docker images
-      run: |
-        ./dockerfiles/build.sh 
+      run:
+          ./dockerfiles/build.sh
```

Comment:
maybe the `|` is still required ?

Reply:
this is not a multiline anymore, so it should work, just as the command above it